Gemfile

```diff
@@ -15,6 +15,8 @@ gem "uglifier", "~> 4.1"
 
 gem "faker", "~> 1.8"
 
+gem "sassc", "~>1.11.0"
```

Comment:
If we plan to deal with this breaking change here, I think we need to restrict this via `gemspecs` so that new applications won't break by default. But I would probably remove the locking from this PR and deal with it separately, so that it's easier to backport.